conscious/chain.json

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "cvn",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://cvn.io/",
+  "pretty_name": "Cvn",
+  "chain_id": "canto_2032-1",
+  "bech32_prefix": "cvn",
```

Comment:
Please register this prefix:
https://github.com/satoshilabs/slips/blob/master/slip-0173.md